src/backend/data/post.js

```diff
@@ -146,7 +150,7 @@ class Post {
       article.guid,
       feed
     );
-    await Promise.all([post.save(), indexPost(post.text, post.id)]);
+    await Promise.all([post.save(), indexPost(post)]);
```

Comment:
Reading this file again, I wonder if we should really be doing `indexPost(post)` inside `post.save()` vs. here.  If a post gets update and needs to be save to the db.

It looks like we don't currently do that, but later on we might want to shuffled this around.  cc @manekenpix, @c3ho.

@StellaJung-Student I think this is fine, but I wanted to make a note.